sdk/typespec/typespec_client_core/Cargo.toml

```diff
@@ -58,6 +59,7 @@ reqwest_rustls = [
 test = [] # Enables extra tracing including error bodies that may contain PII.
 tokio = ["tokio/fs", "tokio/sync", "tokio/time", "tokio/io-util"]
 xml = ["dep:quick-xml"]
+decimal = ["dep:rust_decimal"]
```

Comment:
Why is this optional? Is it so big and pervasive it needs to be a feature?

Reply:
Ah, I see @LarryOsterman requested this for good reason: https://github.com/Azure/azure-sdk-for-rust/pull/2790#issuecomment-3071320416